src/pages/attribute-dictionary.js

```diff
@@ -189,15 +189,35 @@ const EventDefinition = memo(
     let filteredAttributes = [];
 
     if (searchedAttribute) {
-      filteredAttributes = event.attributes.filter(({ name }) =>
-        name.toLowerCase().includes(searchedAttribute.toLowerCase())
-      );
+      filteredAttributes = event.attributes
+        .filter(({ name }) =>
+          name.toLowerCase().includes(searchedAttribute.toLowerCase())
+        )
+        .sort((a, b) => {
```

Comment:
Let's abstract this to a callback function just to keep this more DRY

Reply:
🤦 good call

pushed it up